fossunited/www/schedule.py

```diff
@@ -0,0 +1,9 @@
+import frappe
+
+
+def get_context(context):
+    event_permalink = frappe.form_dict.get("event_permalink")
+
+    frappe.local.flags.redirect_location = f"/dashboard/schedule/{event_permalink}"
```

Comment:
This redirect kind of isn't working: 

![route-schedule-test](https://github.com/user-attachments/assets/a58bde23-f0f5-4e4d-9d42-e27eb35ea0f1)

PS: Let me know if it's not how to be accessed.

Reply:
You need to build for production and then test, `yarn build`

Reply:
Would be good to add it in the PR description as additional context.

Reply:
Not relevant. You are using Port 8000, I think it should be apparent it would not redirect to an app running on port 8080 unless you build it for production. I am treating this as common knowledge 😅

Reply:
Did that - and started bench again - but it's still the same: 
![Uploading Screenshot 2024-08-28 at 3.06.06 PM.png…]()

Reply:
Please check your hooks file and schedule.py file in www folder

Reply:
[Screencast from 2024-08-28 15-16-55.webm](https://github.com/user-attachments/assets/57d73ffb-d810-47ff-a71a-8351cd46f483)

Reply:
I am still unable to see it working properly - if the link is observered - it doesn't also give the /dashboard in the redirect as well. 

![route-schedule-test](https://github.com/user-attachments/assets/40b62cef-0f32-400b-a55c-fd872ed629dd)

Check on this - not looking into this more.

Reply:
![image](https://github.com/user-attachments/assets/ae2bec78-06c9-4d71-b8a3-c3d2753e8d67)

The schedules page worked fine on my local.

Reply:
[screecord-schedules-page.webm](https://github.com/user-attachments/assets/012367c8-b1e1-4d70-802d-978a1dc10899)

It works, lets get this moving!